src/scripts/maker.js

```diff
@@ -580,6 +580,14 @@ class ButtonAction extends EventTarget {
         if (!force && this.disabled) return;
         this.dispatchEvent(new CustomEvent("invoke"));
     }
+
+    detach() {
+        this.buttons.forEach((button) => {
+            button.removeEventListener("click", this.clickListener);
+        });
+        this.setDisabled(false);
```

Comment:
this ought to be `this.disabled = false;`

Reply:
In this case I was thinking that, since the ButtonAction manages the disabled state of its buttons, it should set them to their default disabled states before detaching.  Then a replacement ButtonAction can assume that the buttons are at their defaults when it is added.  Otherwise the ButtonAction constructor would need explicitly set the button's enabled states.

Reply:
... to be sure they match it's default.

Reply:
well i mean there's no such thing as `setDisabled` in this class right? i think setting the property does what you meant

Reply:
Ough.  Yep.  Fixed.